jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml

```diff
@@ -0,0 +1,185 @@
+- job-template:
+    name: '{crowbar_job}'
+    project-type: pipeline
+    disabled: '{obj:disabled|False}'
+    concurrent: '{concurrent|False}'
+
+    logrotate:
+      numToKeep: -1
+      daysToKeep: 30
+
+    triggers: '{triggers}'
+
+    parameters:
```

Comment:
It is probably easy to deduplicate some of these parameters.

Reply:
If you have any ideas about how we can do this, they are most welcome :-)